packages/databricks-vscode/src/extension.ts

```diff
@@ -89,6 +91,7 @@ export async function activate(
         return undefined;
     }
 
+    const workspaceUri = workspace.workspaceFolders[0].uri;
```

Comment:
What if I add several folders to the workspace, all containing databricks bundle configs? Not sure how common that might be, but I personally like to add several folders to the same workspace instead of having multiple VSCode windows open.

I can imagine our side panel you will also need a project selector at the top or something. I wonder if this is something we even want to support in the first iteration.

Reply:
Right now, no. We do not support multiple workspaces. We might look into it later. The option is mostly open to us in this design.

Reply:
The only concern is how much code we will have to re-write. I still don't have a full grasp on all the modules, but it doesn't seem that big of a deal. I guess data providers will change the most, but the rest of the modules will just be instantiated multiple times. Do we have some "global" singleton dependencies that everyone depend on?

Reply:
We do not have any global singleton dependencies except a `WorkspaceConfigs` object which handles vscode configs. It can remain global since it doesn't have any state.